usr/share/whonix-libvirt/xml/Hardened-Debian.xml

```diff
@@ -62,14 +64,15 @@
       <clipboard copypaste='no'/>
       <filetransfer enable='no'/>
     </graphics>
-    <sound model='ich6'/>
+    <sound model='ich6'>
+      <codec type='output'/>
+    </sound>
     <video>
       <model type='qxl' ram='65536' vram='65536' heads='1'/>
     </video>
     <memballoon model='none'/>
     <rng model='virtio'>
-      <rate bytes='1024' period='1000'/>
-      <backend model='random'>/dev/random</backend>
+      <backend model='random'>/dev/urandom</backend>
```

Comment:
This.

(Will post in a comment in forums.)

Reply:
For reference:
https://forums.whonix.org/t/help-welcome-kvm-development-staying-the-course/166/301